package.json

```
@@ -3,61 +3,55 @@
 	"version": "0.1.2",
 	"description": "ESLint configs used at Monogram",
 	"keywords": [
-		"eslint",
-		"eslint-config-airbnb",
-		"eslint-config-next"
+		"eslint-config",
+		"nodejs",
+		"nextjs"
 	],
 	"license": "MIT",
 	"scripts": {
-		"update-local": "pnpm build && pnpm install-locally",
-		"install-locally": "npm install -g",
+		"postinstall": "pnpm build",
+		"build": "tsc",
 		"prepublish": "pnpm build && pnpm release",
-		"build": "tsc --project tsconfig.json",
-		"prestart": "pnpm build",
-		"start": "ts-node src/index.ts",
-		"release": "standard-version",
-		"postinstall": "pnpm build"
-	},
-	"main": "dist/index.js",
-	"bin": {
-		"eslint-config": "dist/index.js"
+		"release": "standard-version"
```

we should add the `prepare` script for Husky to work for people using the repo for the first time
```suggestion
		"release": "standard-version",
		"prepare": "husky install"
```